reporting.md

```diff
@@ -234,7 +234,8 @@ Aggregate reports contain the following fields:
 
 Note that the failure types are non-exclusive; an aggregate report may contain
 overlapping `counts` of failure types when a single send attempt encountered
-multiple errors.
+multiple errors.  Additionally, in the event that only one policy is being 
```

Comment:
I can't parse this last sentence. don't you mean if *multiple* policies are being reported?

Reply:
No, I meant to say that even if there is only one, please still include the array of "policy" inside of "polciies".

Reply:
I don't think it's necessary to say this at all, then. If it's an array, it should always be an array. Isn't that the point of the schema?

Reply:
Yes, but I didn't want a reporter to think it was okay to omit it if the length was one.